src/components/myCohortTeachers/index.js

```diff
@@ -11,7 +11,7 @@ const MyTeachers = ({ teachers }) => {
       <h3 className="my-cohort-teachers--header">{t("teachers")}</h3>
       <ul className="my-cohort-teachers--list">
         {teachers.map((user, idx) => (
-          <UserCard user={user} key={idx} />
+          <UserCard user={user.user.profile} key={idx} />
```

Comment:
I'd probably call the iterated variable `teacher` to make it more clear.

Could it happen to not have the `user` key?

Reply:
@Luca-Terrazzan So ideally it'd be teacher.profile ?

Reply:
`teacher.user.profile` makes sense, but we need to handle what can happen if the `user` key is undefined or does not contain a `profile`; unless you make sure somewhere else that this case can never happen